contracts/rmrk/multiresourcenft/manifest.json

```diff
@@ -0,0 +1,1114 @@
+        {
+          "name": "Is approved",
+          "description": "Action type. If the flag is set to `true` the operator is being granted the rights to manage the tokens and if it is set to `false` the operator is revoked of the rights to manage the tokens.",
+          "recommendedTypes": []
+        }
+      ],
+      "returnDecorators": [],
+      "emittableEvents": ["ApprovalForAll(address,address,bool)"]
+    },
+    {
+      "signature": "setApprovalForAllForResources(address,bool)",
+      "name": "Set approval for all",
+      "description": "Calling this function, caller can choose to give (or remove) rights to operator (first parameter) to manage all of the resources of this ERC721 collection. Caller can grant (or remove) rights to multiple operators by repeatedly calling this function.",
+      "parameterDecorators": [
+        {
+          "name": "Operator",
+          "description": "The account address to get (or loose) rights to manage all of the resources.",
+          "recommendedTypes": ["traits.contractCaller"]
+        },
+        {
+          "name": "Is approved",
+          "description": "Action type. If the flag is set to `true` the operator is being granted the rights to manage the resources and if it is set to `false` the operator is revoked of the rights to manage the resources.",
+          "recommendedTypes": []
+        }
+      ],
+      "returnDecorators": [],
+      "emittableEvents": ["ApprovalForAllForResources(address,address,bool)"]
+    },
+    {
+      "signature": "setFallbackURI(string)",
+      "name": "Sets fallback metadata URI",
+      "description": "Metadata URI",
+      "parameterDecorators": [
+        {
+          "name": "Fallback metadata URI",
+          "description": "Metadata URI",
+          "recommendedTypes": []
+        }
+      ],
+      "returnDecorators": [],
+      "emittableEvents": []
+    },
+    {
+      "signature": "setLock(bool)",
+      "name": "Set lock",
+      "description": "Sets lock on the contract",
+      "parameterDecorators": [
+        {
+          "name": "Lock value",
+          "description": "Boolean value",
+          "recommendedTypes": []
+        }
+      ],
+      "returnDecorators": [],
+      "emittableEvents": []
+    },
+    {
+      "signature": "setPriority(uint256,uint16[])",
+      "name": "Set priority",
+      "description": "Sets priorities for the given token id",
+      "parameterDecorators": [
+        {
+          "name": "Token ID",
+          "description": "ID of the token transferred",
+          "recommendedTypes": []
+        },
+        {
+          "name": "Priorities",
+          "description": "Array of priroities",
+          "recommendedTypes": []
+        }
+      ],
+      "returnDecorators": [],
+      "emittableEvents": []
+    },
+    {
+      "signature": "setTokenEnumeratedResource(uint64,bool)",
+      "name": "Set Token Enumerated Resource",
+      "description": "Sets token enumerated resource state",
```

Comment:
Sets token enumerated resource state. If this is active for a resource, the tokenId is appended when getting TokenURI for it.